### ISSUE TITLE
antismash-hmmscan runtime error

### ISSUE THREAD

**Issue by alienzj** [antismash/antismash#204]
**Describe the bug**
antismash 5.0 runntime error:
RuntimeError: hmmscan returned 1: 'Error: TC bit thresholds unavailable on model Enediyne-KS' while scanning '>CMLKBGAL_01274\nMSSKLIYTGKAKDIYTTEDEHVIRSVYKDQATMLNGARKETIEGKGVLNNQISSLIFEKLNAAGVATHFIERISDTEQLNKKVT'

**Expected behavior**

**To Reproduce**
For analysis issues, provide an accession number or a record fragment that can reproduce the problem.
```bash
antismash --cpus 8 --cf-create-clusters --smcog-trees --cb-knownclusters --asf --pfam2go --output-dir results/umgs/ERR2764806.metaspades.bin.4.fa --genefinding-gff3 ERR2764806.metaspades.bin.4.gff ERR2764806.metaspades.bin.4.fna
```

For interaction issues, also include steps to reproduce the behavior:

**Screenshots**
![image](https://user-images.githubusercontent.com/8847420/66460567-c4954400-eaa9-11e9-96ea-5a55319056d6.png)

**System (please complete the following information):**
 - OS: [e.g. iOS, Ubuntu]
centos 6.9
 - Browser [e.g. chrome, safari] (if relevant)
chromium 77.0.3865.90

**Additional context**

**Comment by SJShaw** [comment]
This was fixed in 8053aa55112159a4d74981d430f65f47750c07b5 and will be included in the next release.

**Comment by alienzj** [comment]
I will test it. Thanks~
